test/run-spec-wasm2c.py

```diff
@@ -537,6 +537,9 @@ def main(args):
     parser.add_argument('--disable-bulk-memory', action='store_true')
     parser.add_argument('--disable-reference-types', action='store_true')
     parser.add_argument('--debug-names', action='store_true')
+    parser.add_argument('-j', '--num-outputs', metavar='COUNT',
```

Comment:
I think whatever we call this option it should be the same for both `run-spec-wasm2c.py` and `wasm2c` itself.   I actually like `--num-outputs` if you want to go with that.   Perhaps we can just skip the short arg version?